src/vfs/public.c

```diff
@@ -48,12 +50,17 @@ bool vfs_unmount(const char *path) {
 	return false;
 }
 
+
+//not absolute path(this path does not have zip)
 SDL_RWops* vfs_open(const char *path, VFSOpenMode mode) {
 	SDL_RWops *rwops = NULL;
 	char p[strlen(path)+1];
-	path = vfs_path_normalize(path, p);
+	
+    //This second line is similar to 'vfs_repr' in vfs/public.c
+    path = vfs_path_normalize(path, p);
```

Comment:
This comment is misleading; `vfs_repr` and `vfs_path_normalize` do fundamentally different things. 

`vfs_repr` returns a human-readable string with information about a VFS path suitable for printing out, e.g. a "real" system path that the VFS node corresponds to, or a path that points inside of a zip-file. Its output is intended for diagnostics only.

`vfs_path_normalize` takes a VFS path and converts it into its canonical form. The result is always a VFS path that points to the same object as the original path. For example, it will convert `/foo///bar/../example` and `foo/./example` both to `foo/example`. Unfortunately, the guarantees of this canonical form are not yet documented.